SwiftGit2/Repository.swift

```diff
@@ -185,31 +185,31 @@ final public class Repository {
 	///
 	/// Returns the result of calling `transform` or an error if the object
 	/// cannot be loaded.
-	func withLibgit2Object<T>(_ oid: OID, type: git_otype, transform: (OpaquePointer) -> Result<T, NSError>) -> Result<T, NSError> {
+	private func mapGitObject<T>(withIdentity oid: OID, type: git_otype, transform: (OpaquePointer) -> Result<T, NSError>) -> Result<T, NSError> {
```

Comment:
I think this would be better to remain as `with*`. It's not exactly a `map` since it's loading the object, not just transforming it.